package.json

```diff
@@ -65,7 +65,8 @@
     "react": "^16.0.0",
     "react-dom": "^16.0.0",
     "string-punctuation-tokenizer": "^2.0.0",
-    "word-aligner": "^0.3.0"
+    "word-aligner": "^0.3.0",
+    "usfm-js": "^2.1.0"
```

Comment:
Word-aligner would need to be updated right ?

Reply:
it's a peer dependency, but I can do that once word-aligner is published.